src/khoj/interface/web/index.html

```diff
@@ -94,9 +94,12 @@
                 setQueryFieldInUrl(query);
 
             // Generate Backend API URL to execute Search
-            url = type === "image"
-                ? `/api/search?q=${encodeURIComponent(query)}&t=${type}&n=${results_count}&client=web`
-                : `/api/search?q=${encodeURIComponent(query)}&t=${type}&n=${results_count}&r=${rerank}&client=web`;
+            if (type == 'all')
```

Comment:
nit: Can use a helper function + a string constant for the URL to create these urls. For example:

```
function createRequest(query, n_count, content_type, rerank) {
    url = `/api/search?q=${encodeURIComponent(query)}&n=${results_count}`;
    if (content_type):
        url += `&t={type}`;
    // ...add on other query parameters as required.
    return url;
}
```